manifests/init.pp

```diff
@@ -32,7 +32,7 @@
     default: { fail("ERROR: ${::osfamily} based systems are not supported!") }
   }
 
-  package { $pkg_setuptools: ensure => installed, }
+  package { $pkg_setuptools: ensure => $::operatingsystemmajrelease ? { '7' => 'latest', default => '1.3.1-4.el6.art'} }
```

Comment:
The line above is already in puppet, I'm simply trying keep the tree up to date.